src/main/webapp/js/studentProfile.js

```diff
@@ -62,9 +62,7 @@ function finaliseUploadPictureForm(event) {
         	$('#statusMessage').css("display", "block")
         	        .attr('class', 'alert alert-danger')
         	        .html('There seems to be a network error, please try again later');
-        	$("html, body").animate({
-        		scrollTop: 0
-        	});
+        	scrollToTop({duration: ''});
```

Comment:
The 'fast', 'slow', any other string behaviour is inherited from jQ's animation function.
> The strings 'fast' and 'slow' can be supplied to indicate durations of 200 and 600 milliseconds, respectively. If any other string is supplied, or if the duration parameter is omitted, the default duration of 400 milliseconds is used.

As I used the function for instant scrolling as well, I used `undefined` (omission of parameter) for "no animation" and as per jQ, a string / number for "animation".

Not sure if it's a good idea to type an unnecessary and nonstandard `'normal'` in here just for that. It would result in the same speed as `'super-duper-fast'`, for example.